configs/cascade_rcnn/cascade_mask_rcnn_x101_64x4d_fpn_mstrain_3x_coco.py

```diff
@@ -0,0 +1,14 @@
+_base_ = './cascade_mask_rcnn_r50_caffe_fpn_mstrain_3x_coco.py'
```

Comment:
Should not based on caffe style. `img_norm_cfg` is different.

Reply:
You found the blind spot！